app/controllers/admin/users_controller.rb

```diff
@@ -37,11 +37,11 @@ def edit
     if params[:edit]
       redirect_to edit_admin_user_path(params[:edit][:user_id]) and return
     end
-    @user = User.find params[:id]
+    @user = User.where(:active => true).find params[:id]
   end
 
   def update
-    @user  = User.find params[:id]
+    @user  = User.where(role: 0).find params[:id]
```

Comment:
This works, but I'd prefer `User.where(role: "pcv")` or `User.pcv`. `role: 0` is fairly inscrutable.